java/kotlin-extractor2/src/main/kotlin/entities/Expression.kt

```diff
@@ -223,6 +224,61 @@ private fun KotlinFileExtractor.extractExpression(
                 // We're handling it in the children with the below
                 extractExpression(e.baseExpression!!, callable, parent)
             }
+
+            is KtIsExpression -> {
+
+                val locId = tw.getLocation(e)
+                val type = useType(e.expressionType!!)
+                val exprParent = parent.expr(e, callable)
+
+                val id: Label<out DbExpr>
+                if (e.isNegated) {
+                    id = tw.getFreshIdLabel<DbNotinstanceofexpr>()
+                    tw.writeExprs_notinstanceofexpr(id, type.javaResult.id, exprParent.parent, exprParent.idx)
+                } else {
+                    id = tw.getFreshIdLabel<DbInstanceofexpr>()
+                    tw.writeExprs_instanceofexpr(id, type.javaResult.id, exprParent.parent, exprParent.idx)
+                }
+
+                tw.writeExprsKotlinType(id, type.kotlinResult.id)
+                extractExprContext(id, locId, callable, exprParent.enclosingStmt)
+                extractExpressionExpr(e, callable, id, 0, exprParent.enclosingStmt)
```

Comment:
Shouldn't `e` here be `e.getLeftHandSide()`?

Reply:
🤦 Ahh, thanks for finding that.